packages/std/src/lib.rs

```diff
@@ -45,6 +45,7 @@ pub use crate::ibc::{
     IbcDestinationCallbackMsg, IbcDstCallback, IbcEndpoint, IbcMsg, IbcOrder, IbcPacket,
     IbcPacketAckMsg, IbcPacketReceiveMsg, IbcPacketTimeoutMsg, IbcReceiveResponse,
     IbcSourceCallbackMsg, IbcSrcCallback, IbcTimeout, IbcTimeoutBlock, IbcTimeoutCallbackMsg,
+    TransferMsgBuilder,
```

Comment:
Do we need the trait here too?

Reply:
Only if we want users to implement / use it, which is not the case here, I think.
However, we are also not exporting the memo source implementations right now.
This means, we can be more flexible with those in the future (e.g. renaming them, changing the generics of the builder type), but the drawback of not exporting them is that users have no way of specifying the full type, so the builder can only be used where type inference works. This shouldn't be *too* much of a problem for a builder, but it does prevent starting a builder in one function and passing it on to another, so it might be nicer to export them.
In that case, do we keep them in a sub-module? Seems a bit weird to export something like `EmptyMemo` at the top level.

Reply:
Thank. Let's keep it private then. We can always open up later. The real use case would be support for other middlewares that have their own data structure. But that's a problem when it's a problem.